src/agent/rustjail/src/mount.rs

```diff
@@ -1401,6 +1408,91 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_parse_mount_table() {
+        #[derive(Debug)]
+        struct TestData<'a> {
+            mountinfo_data: Option<&'a str>,
+            result: Result<Vec<Info>>,
+        }
+
+        let tests = &[
```

Comment:
Could you add a few more tests to this list. Specfiically:

- A test where the first fields are valid, but which has too many optional fields in it.
- Tests where `mountinfo_data` is `"-"`, `"--"`, `"- -"` and `" - "`.

Reply:
What would too many optional fields look like? I was looking at [this documentation](https://www.kernel.org/doc/Documentation/filesystems/proc.txt), which says "zero or more optional fields."

Reply:
Again, I meant "final fields" here, not "optional fields". There should be 3 fields after the optional field terminator (`-`).

Reply:
One other thought: it might be worth adding a test for a mount that contains the pattern `" - "`. This isn't documented in `proc(5)`, but the spaces get encoded as shown below:

```bash
$ mkdir -p /tmp/"foo - bar"
$ sudo mount -obind /tmp/"foo - bar" /mnt
$ sudo grep foo /proc/self/mountinfo
1537 94 0:39 /foo\040-\040bar /mnt rw,nosuid,nodev shared:58 - tmpfs tmpfs rw,seclabel,size=16324552k,nr_inodes=1048576,inode64
```